src/legacy/core_plugins/kibana/public/index.scss

```diff
@@ -37,9 +37,23 @@
 // MUST STAY AT THE BOTTOM BECAUSE OF DARK THEME IMPORTS
 @import './dashboard/index';
 
-// helper class that hides elements when rendered in percy
+// helper classes to show/hide elements in Percy
 @media only percy {
   .hideInPercy {
     visibility: hidden;
+
+    img {
+      display: none;
```

Comment:
This seems suspicious. Just testing something?

Reply:
Ya I tried to hide the `discoverChart` and it was still visible.

I think it has to do with the `canvas`/`img` swap logic. But this didn't fix it 😞

Reply:
Any ideas?